src/components/Heading/Heading.module.css

```diff
@@ -13,11 +13,11 @@
   --heading-font-weight: var(--eds-typography-font-weight-normal);
   --heading-font-family: var(--eds-typography-font-family-1);
   --heading-letter-spacing: var(--eds-typography-letter-spacing-normal);
-  --heading-text-transform: ;
 
   font: unset;
   font-family: var(--heading-font-family);
   font-size: calc(var(--heading-font-size) * 1px);
+  font-style: var(--heading-font-style, initial);
   line-height: calc(var(--heading-line-height) * 1px);
   font-weight: var(--heading-font-weight);
```

Comment:
Before , i was setting a default value of ` ` to the CSS variable, which prevented the fallback from being seen and used, removing this line means it will properly use the fallback values for both `*-font-style` and the previous `*-text-transform`